projects/gui/src/mainwindow.cpp

```diff
@@ -159,6 +159,10 @@ void MainWindow::createActions()
 
 	m_flipBoardAct = new QAction(tr("&Flip Board"), this);
 
+	m_adjudicateGameAct2 = new QAction(tr("Ad&judicate Draw"), this);
```

Comment:
Could you rename these to `m_adjudicateDrawAct`, `m_adjudicateWhiteWinAct` and `m_adjudicateBlackWinAct`?

Reply:
yes, of cause this is clearer